salt/_pillar/metalk8s_nodes.py

```diff
@@ -92,9 +96,40 @@ def ext_pillar(minion_id, pillar, kubeconfig):
             for node in node_list.items
         )
 
+        custom_object = []
```

Comment:
`volume_information` make more sense IMO as you not store all "custom objects" in this dict